Homepage.jsx

```diff
@@ -130,15 +130,15 @@ export default function HomePage() {
                   </li>
                   <li>
                     <div className={styles.checklistIcon} />
-                    <p>Manage users' accounts</p>
+                    <p>Manage users&apos; accounts</p>
                   </li>
                   <li>
                     <div className={styles.checklistIcon} />
                     <p>Monitor existing contracts</p>
                   </li>
                   <li>
                     <div className={styles.checklistIcon} />
-                    <p>View users' historical transactions</p>
+                    <p>View users&apos; historical transactions</p>
```

Comment:
I think we should still be using a single quote(') which will improve readability. What's preventing us from using it?